queries.sql

```diff
@@ -6,3 +6,29 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * from animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Vet clinic database: query and update animals table
+BEGIN;
+UPDATE animals SET species='unspecified';
+ROLLBACK;
+UPDATE animals SET species='digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species='pokemon' WHERE name LIKE IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;

```

Comment:
- [ ] Great work on your query to delete all records from the animal table! 👏 However, as a best practice, it's crucial to verify the success of your changes after each transaction. To ensure that the rollback operation effectively undoes the changes, I recommend querying the animal table following the rollback. This step adds an extra layer of confidence in the integrity of your database.

   - By executing a simple query to retrieve the data from the animal table after the rollback, you can confirm whether the changes were properly rolled back and that the data remains consistent.
   
Happy coding! 😊🚀🍀